sdb/commands/zfs/btree.py

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright 2019 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+from typing import Iterable
+
+import drgn
+import sdb
+
+
+class Btree(sdb.Walker):
+    """
+    walk zfs_btree
+
+    DESCRIPTION
+
+        btree_t's are used in ZFS, especially as metaslab range trees. Use walk or
+	zfs_btree for an in-order traversal of the contents of the tree.
+
+    EXAMPLES
+
+        > <metaslab_t *> | member ms_allocatable | walk | cast range_seg32_t *
+
+        *(range_seg32_t *)0xffff9990b5141d68 = {
+	.rs_start = (uint32_t)862981,
+	.rs_end = (uint32_t)868158,
+	}
+	*(range_seg32_t *)0xffff9990b5141d70 = {
+	.rs_start = (uint32_t)868163,
+	.rs_end = (uint32_t)939419,
+	}
+	*(range_seg32_t *)0xffff9990b5141d78 = {
+	.rs_start = (uint32_t)939424,
+	.rs_end = (uint32_t)1048576,
+	}
```

Comment:
Out of curiosity do the tabs look aligned when the help message is printed?
From the code it seems like the first line in the description section is not aligned with the second one, similarly for the lines in the example section.

Reply:
It looks fine in the output, but yeah, there's a tabs vs spaces issue. I'll update the PR.